crates/node/ops/fs.rs

```diff
@@ -78,3 +79,146 @@ where
     fs.cp_async(path, new_path).await?;
     Ok(())
 }
+
+#[derive(Debug, Serialize)]
+pub struct StatFs {
+    #[serde(rename = "type")]
+    pub typ: u64,
+    pub bsize: u64,
+    pub blocks: u64,
+    pub bfree: u64,
+    pub bavail: u64,
+    pub files: u64,
+    pub ffree: u64,
+}
+
+#[op2]
+#[serde]
+pub fn op_node_statfs<P>(
```

Comment:
I guess this should be a no-op? We don't expose Deno's FS API too.

Reply:
Sure